### ISSUE TITLE
EssentialsX GeoIP no work

### ISSUE THREAD

**Issue by GreatGoal0** [EssentialsX/Essentials#5505]
### Type of bug

Other unexpected behaviour

### `/ess dump all` output

In a moment I edit and add 

### Bug description

%essentials_geolocation% Returns empty message and /whois with permission Essentials.geoip.show and all config functions to true only return player info and IP, nothing of city and locality gives me back the world  

### Steps to reproduce

/papi parse me %essentials_geolocation%
/whois playername

### Expected behaviour

%essentials_geolocation% returns the correct value 
/whois playername the player's city/locate appears 

### Actual behaviour

%essentials_geolocation% Returns empty message and /whois with permission Essentials.geoip.show and all config functions to true only return player info and IP, nothing of city and locality gives me back the world

**Comment by pop4959** [comment]
Please report this issue to the author of the addon you installed which adds these placeholders. Essentials does not include or maintain any of the PlaceholderAPI expansions itself.

**Comment by GreatGoal0** [comment]
But if EssentialsX GeoIP is from EssentialsX

**Comment by GreatGoal0** [comment]
Why the creator of EssentialsX is also faith EssentialsX GeoIP

**Comment by pop4959** [comment]
Yes Essentials maintains GeoIP, but the placeholder you mentioned is not part of Essentials.

**Comment by GreatGoal0** [comment]
That's why!! Read the message again, Essentials GeoIP does not work

**Comment by Evidentsinger14** [comment]
Please provide actual information, and it can be investigated. 90 percent of what you said was about a placeholder, and you still have not given a dump link.

**Comment by mbax** [comment]
Your report is about a placeholder. The placeholder is **not** Essentials or EssentialsGeoIP. It *talks* to it, but you should start with the provider of the placeholder (which is *not* EssentialsX)

**Comment by pop4959** [comment]
Essentials GeoIP definitely works. Maybe you didn't configure your access token?

If you need help configuring Essentials GeoIP join the [Discord server](https://discord.gg/PHpuzZS).

**Comment by GreatGoal0** [comment]
The placeholder has no errors is well done, it returns an empty message why you didn't get the EssentialsX GeoIP information, /whois with the essentials.geoip.show permission and the show-cities option and all the others in config.yml are true and only return information about the player and his ip, nothing about the city and the town appears that of the world

**Comment by GreatGoal0** [comment]
I already configured the maxmind key, in fact there are no errors. start and if you download the .mmdb database

**Comment by GreatGoal0** [comment]
I tried to ask for help on EssentialsX discord but it was useless with the support they have alas

**Comment by 0372JesusMtz0372** [comment]
> Proporcione información real y podrá ser investigada. El 90 por ciento de lo que dijiste fue sobre un marcador de posición y aún no has proporcionado un enlace de volcado.



> Proporcione información real y podrá ser investigada. El 90 por ciento de lo que dijiste fue sobre un marcador de posición y aún no has proporcionado un enlace de volcado.

Damn friend, the information of the issue is on the placeholder and /whois, take a good look

**Comment by mbax** [comment]
> Damn friend, the information of the issue is on the placeholder

Yes. Ask the placeholder author for help.

Still waiting on your dump, as you said you would add.

**Comment by pop4959** [comment]
I understand the frustration, although at the moment this issue is still invalid since it involves an add-on we don't maintain, and the claim that location information is not shown in /whois is also not reproducible.

We will not be able to help you much better here than on Discord, so I'd advise continuing to troubleshoot there. The only thing posting here accomplishes is spamming everyone who has subscribed to email notifications.

If you can provide valid steps to reproduce your issue with /whois and *without any other plugins installed* feel free to open a new issue. Thank you.
